hack/ci/testdata/kubermatic_dualstack.yaml

```diff
@@ -24,6 +24,10 @@ spec:
   imagePullSecret: '__IMAGE_PULL_SECRET__'
   userCluster:
     apiserverReplicas: 2
+    operatingSystemManager:
+      imageTag: a73915c8502e4f5b1728bb9c7e438393336dc846
+    machineController:
+      imageTag: d24cab926eb21dd7545aafa19ae421f9896b2cba
```

Comment:
We don't need to pin these images any more as https://github.com/kubermatic/kubermatic/pull/10856 will update KKP to use the latest available images

Reply:
I know. Was waiting for the release. Can change now that we have them.